app/lib/helpers/reportStatusHelpers.tsx

```diff
@@ -0,0 +1,36 @@
+import { DateTime } from "luxon";
+
+export const daysUntilReportDue = (reportDueDate) => {
```

Comment:
can we type those date parameters, not sure if we're expecting a `string`, `Date` or luxon `DateTime`